code/modules/mob/living/simple_animal/hostile/megafauna/_megafauna.dm

```diff
@@ -70,6 +70,11 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed, list/force_grant)
+	if(gibbed) // in case they've been force dusted
+		SEND_SIGNAL(src, COMSIG_LIVING_DEATH, gibbed)
+		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_MOB_DEATH, src, gibbed)
+		return
```

Comment:
I do not like signals being sent from more than one location, is there some reason you can't fix this while calling the parent proc?

Reply:
Big maybe - it's pretty messy, looks like they built this to prevent exploiting to get loot or achievements

Reply:
added a solution where it's calling parent. not sure if this is what you were looking for - normally it does not call parent at all in this case, but i have it to where it's behaving as before